src/components/Card/Card.jsx

```diff
@@ -250,6 +255,8 @@ const Card = ({
       )}
     </VisibilitySensor>
   );
+
+  return isExpanded ? <div className="bx--modal is-visible">{card}</div> : card;
```

Comment:
The `bx` prefix is user configurable, we'll need to import and use the prefix variable from `settings`. We do this in `Button.jsx` as an example

Reply:
thanks for catching, will fix